xbmc/cores/paplayer/VideoPlayerCodec.cpp

```diff
@@ -529,3 +529,16 @@ CAEStreamInfo::DataType VideoPlayerCodec::GetPassthroughStreamType(AVCodecID cod
   else
     return CAEStreamInfo::DataType::STREAM_TYPE_NULL;
 }
+
+int VideoPlayerCodec::GetData(uint8_t **data)
```

Comment:
this wrapper produces a lot of overhead. for every read it creates an object on the stack and copies data to/from it.

Reply:
would be DVDFrame member an option?